api/v4/source/logs.yaml

```
@@ -0,0 +1,18 @@
+/api/v4/logs/download:
+  post:
```

The one CI check is failing because it the type for this is incorrect
```suggestion
  get:
```